### PR TITLE
Make SSV validate PF content mailboxes

### LINKED ISSUE #829: [Issue] SourceSideValidations - Should validate ContentMailboxName and ContentMailboxGuid
We had a long-running case due to ContentMailboxName and ContentMailboxGuid being blank or invalid on some folders. These should be part of the SSV checks.

### PR DESCRIPTION
**Pull request by bill-long**
Fixes #829 and #1922.

### CODE REVIEW

**Comment by dpaulson45** [comment]
FYI this won't auto close both issues.

File: PublicFolders/SourceSideValidations/SourceSideValidations.ps1 [hunk 1]
```diff
@@ -117,29 +117,48 @@ try {
...
+        return
+    }
+
+    $folderData.MailboxToServerMap = $mailboxToServerMap
+
+    # Validate that all content mailboxes exist
+    $ipmSubtreeByMailboxGuid = $folderData.IpmSubtree | Group-Object ContentMailboxGuid
+    foreach ($group in $ipmSubtreeByMailboxGuid) {
+        if ([string]::IsNullOrEmpty($group.Name)) {
+            Write-Warning "$($group.Count) folders have no ContentMailboxGuid. "
```
Comment (dpaulson45) [inline PublicFolders/SourceSideValidations/SourceSideValidations.ps1:152]: Is this a point to attempt to get the mailbox information after this point if the name is in fact null or empty?

Just trying to understand the reason for this `if` statement as the result will come back with a `$null` value and provide you with the same information on line 156.

Comment (bill-long) [reply PublicFolders/SourceSideValidations/SourceSideValidations.ps1]: Good point. I added that when I was trying to produce a null value, but I was never able to actually do that. I think we can just let it fall through to the existing logic.

**Review by dpaulson45 — APPROVED** [review]
Due to new issues coming up regarding what this fix, we can't wait anymore on the test team. It has been over 3 weeks since this code has been ready for production.

**Pull request merged**